crop-me/src/main/AndroidManifest.xml

```diff
@@ -1,31 +1,31 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mikelau.croperino">
+<manifest package="com.mikelau.croperino"
+          xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <permission android:name="${applicationId}.provider.READ" />
-    <uses-permission android:name="${applicationId}.provider.READ"/>
+  <permission android:name="${applicationId}.provider.READ"/>
+  <uses-permission android:name="${applicationId}.provider.READ"/>
 
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
+  <application
+      android:allowBackup="true"
+      android:label="@string/app_name"
+      android:supportsRtl="true">
 
-        <activity
-            android:name=".CropImage"
-            android:configChanges="keyboardHidden|orientation"
-            android:label="@string/app_name"
-            android:screenOrientation="portrait"
-            />
+    <activity
+        android:name=".CropImage"
+        android:configChanges="keyboardHidden|orientation"
+        android:label="@string/app_name"
+        android:screenOrientation="portrait"
+        />
 
-        <provider
-            android:name="android.support.v4.content.FileProvider"
-            android:authorities="${applicationId}.provider"
-            android:exported="false"
-            android:grantUriPermissions="true"
-            android:readPermission="${applicationId}.provider.READ">
-            <meta-data
-                android:name="android.support.FILE_PROVIDER_PATHS"
-                android:resource="@xml/provider_paths"/>
-        </provider>
+    <provider
+        android:name="com.mikelau.croperino.FileProvider"
```

Comment:
For projects that have their own default file provider defined, it will cause a conflict, so use a custom file provider here.